tests/commands_test.py

```diff
@@ -3,11 +3,11 @@
 
 from mentat.code_context import CodeContext
 from mentat.commands import (
-    AddCommand,
     Command,
+    ExcludeCommand,
     HelpCommand,
+    IncludeCommand,
```

Comment:
We were using `Add`/`Remove` to mean different things in different places. Now it's `include` to add something to the context sent to gpt, and `create` to make a new code file in the directory.

Reply:
I'm actually really happy you made this change; it was a bit confusing haha